notebooks/summit/AuxTel/SITCOM-627/SITCOM-627_Measure_Astigmatism.ipynb

```diff
@@ -0,0 +1,1511 @@
+{
```

Comment:
<p>I like this. Just a heads up I mentioned to Craig that you do this and said he should ping you for an example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
👍